template/build/webpack.base.conf.js

```diff
@@ -31,76 +31,73 @@ module.exports = {
       'components': path.resolve(__dirname, '../src/components')
     }
   },
-  resolveLoader: {
-    fallback: [path.join(__dirname, '../node_modules')]
-  },
   module: {
+    rules: [
     {{#lint}}
-    preLoaders: [
       {
         test: /\.vue$/,
-        loader: 'eslint',
+        loader: 'eslint-loader',
+        enforce: 'pre',
         include: [
           path.join(projectRoot, 'src')
         ],
         exclude: /node_modules/
       },
       {
         test: /\.js$/,
-        loader: 'eslint',
+        loader: 'eslint-loader',
+        enforce: 'pre',
         include: [
           path.join(projectRoot, 'src')
         ],
         exclude: /node_modules/
-      }
-    ],
+      },
     {{/lint}}
-    loaders: [
       {
         test: /\.vue$/,
-        loader: 'vue'
+        loader: 'vue-loader'
       },
       {
         test: /\.js$/,
-        loader: 'babel',
+        loader: 'babel-loader',
         include: [
           path.join(projectRoot, 'src')
         ],
         exclude: /node_modules/
```

Comment:
I think it you don't have exclude it works on a whitelist basis

    it('should not match if include does not match', function() {
		var loader = new RuleSet([{
			test: /\.css$/,
			include: /output.css/,
			loader: 'css'
		}]);
		(match(loader, 'style.css')).should.eql([]);
	});